drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-backend/src/main/java/org/drools/workbench/screens/guided/rule/backend/server/GuidedRuleDSLRSourceService.java

```diff
@@ -31,7 +31,6 @@
 import org.drools.compiler.lang.dsl.DSLTokenizedMappingFile;
 import org.drools.compiler.lang.dsl.DefaultExpander;
 import org.drools.workbench.models.commons.backend.rule.RuleModelDRLPersistenceImpl;
-import org.drools.workbench.models.commons.backend.rule.RuleModelIActionPersistenceExtension;
```

Comment:
Unused import: javax.enterprise.inject.Instance;

probably not changed by your changes, but would you mind remove it?